library/src/main/java/org/xmtp/android/library/libxmtp/Member.kt

```diff
@@ -1,11 +1,13 @@
 package org.xmtp.android.library.libxmtp
 
 import uniffi.xmtpv3.FfiGroupMember
+import uniffi.xmtpv3.FfiPermissionLevel
 
 class Member(private val ffiMember: FfiGroupMember) {
-
     val inboxId: String
         get() = ffiMember.inboxId
     val addresses: List<String>
         get() = ffiMember.accountAddresses
+    val permissionLevel: FfiPermissionLevel
```

Comment:
I like to typecast the ffi stuff to a friendly name. Kind of like what I did here: https://github.com/xmtp/xmtp-android/blob/main/library/src/main/java/org/xmtp/android/library/libxmtp/MessageV3.kt#L32-L37

Reply:
updated here https://github.com/xmtp/xmtp-android/pull/245/commits/4fde146ac2d91bac0faee2a5597e59573d90b46c